cask/src/cask/main/Main.scala

```diff
@@ -62,9 +62,16 @@ abstract class Main {
     null
   }
 
+  private def screenExecutor(executor: Executor): Executor = {
+    if (executor eq null) executor
+    else if (System.getProperty("cask.virtualThread.enabled", "true").toBoolean) {
+      Util.createVirtualThreadExecutor(executor).getOrElse(executor)
+    } else executor
+  }
```

Comment:
@lihaoyi Wdyt about this?

Reply:
I will update the PR later, and still need update the CI to Java 21 to run the benchmarks.

Reply:
That's the plan.